source/clear-linux/tutorials/kubernetes.rst

```diff
@@ -221,17 +173,11 @@ the :file:`/etc/crio/crio.conf` file to add:
 
 **Notes about Weave Net add-on**
 
-If you choose the `Weave Net` add-on, then you must make the following changes
-because it installs itself in the :file:`/opt/cni/bin` directory.
-
-If you are using Docker and `Weave Net`, edit the :file:`kubeadm.conf` file to
-add:
-
-..  code-block:: bash
-
-    Environment="KUBELET_NETWORK_ARGS=--network-plugin=cni --cni-conf-dir=/etc/cni/net.d --cni-bin-dir=/opt/cni/bin"
+If you choose the `Weave Net` add-on, then you must make the following
+changes because it installs itself in the :file:`/opt/cni/bin` directory.
```

Comment:
I am pretty sure the default crio.conf now has  plugin_dir = "/opt/cni/bin" which means I don't think it would work for either flannel or weave plugins without some further configuration.  Manohar's script at https://github.com/clearlinux/cloud-native-setup/blob/master/clr-k8s-examples/setup_system.sh provides clues on what users need to do.

I *believe* this command will solve the issue for both Weave and flannel and calico and other network overlays.    Please have Mikko confirm this is the correct final approach as I don't know what other changes to clear bundles might come.

sudo ln -s /usr/libexec/cni /opt/cni/bin

Reply:
Alternatively, like @eadamsintel mentioned, we could have users run Manohar's Bash script. Thoughts?